### ISSUE TITLE
Credentials can be removed while in use

### ISSUE THREAD

**Issue by rjeczalik** [koding/koding#9271]
Despite having built stacks that use a credential, it's possible to remove it, which makes start/stop/apply operations to fail. 

## Expected Behavior

Remove should be disabled for credentials in use.

## Current Behavior

It is not disabled.

## Possible Solution

Disable it.

**Comment by gokmen** [comment]
There is no way to prevent user for doing that, even it's prevented on Koding side it's still possible to remove/disable it from provider side. So, client-side should handle these cases correctly.

**Comment by rjeczalik** [comment]
@gokmen Yes, one can revoke AWS access tokens manually, however it's possible to remove it from Koding UI. I made an record of a repo case, but it's still processing 😕  (http://recordit.co/SvKhLHBWpr)

**Comment by gokmen** [comment]
all I'm saying instead of preventing user to remove credential from Koding side, we need to handle this case on client-side, kloud also needs to provide meaningful errors for this case.
